src/com/owncloud/android/providers/UsersAndGroupsSearchProvider.java

```diff
@@ -288,15 +288,19 @@ public void run() {
                 // The Toast must be shown in the main thread to grant that will be hidden correctly; otherwise
                 // the thread may die before, an exception will occur, and the message will be left on the screen
                 // until the app dies
-                Toast.makeText(
-                        getContext().getApplicationContext(),
-                        ErrorMessageAdapter.getErrorCauseMessage(
-                                result,
-                                null,
-                                getContext().getResources()
-                        ),
-                        Toast.LENGTH_SHORT
-                ).show();
+
+                // Edited: this toast message has no sense. If operation is being passed as null to getErrorCauseMessage(),
```

Comment:
That's true, except when the ```result``` is an error included in ```ErrorMessageAdapter#isNetworkError(...)``` . In that case the value of ```operation``` is not considered at all in ```ErrorMessageAdapter#getErrorCauseMessage(...)``` .